docs/core/tools/dotnet-format.md

```diff
@@ -0,0 +1,146 @@
+ms.date: 07/12/2021
+---
+# dotnet format
+
+**This article applies to:** ✔️ .NET 6.x SDK and later versions
+
+## Name
+
+`dotnet format` - Formats code to match `editorconfig` settings.
+
+## Synopsis
+
+```dotnetcli
+dotnet format [options] [<PROJECT | SOLUTION>]
+
+dotnet format -h|--help
+```
+
+## Description
+
+`dotnet format` is a code formatter that applies style preferences to a project or solution. Preferences will be read from an *.editorconfig* file, if present, otherwise a default set of preferences will be used. For more information, see the [EditorConfig documentation](../../fundamentals/code-analysis/configuration-files.md#editorconfig).
+
+## Arguments
+
+`PROJECT | SOLUTION`
+
+The MSBuild project or solution to run code formatting on. If a project or solution file is not specified, MSBuild searches the current working directory for a file that has a file extension that ends in *proj* or *sln*, and uses that file.
+
+## Options
+
+None of the options below are required for the `dotnet format` command to succeed but can be used to further customize what is formatted and by which rules.
+
+* **`--diagnostics <DIAGNOSTICS>`**
+
+  A space-separated list of diagnostic IDs to use as a filter when fixing code style or third party issues. Default value is whichever IDs are listed in the *editorconfig* file. For a list of built-in analyzer rule IDs that you can specify, see the [list of IDs for code-analysis quality rules](../../fundamentals/code-analysis/quality-rules/index.md).
+
+* **`--severity`**
+
+  The minumum severity of diagnostics to fix. Allowed values are `info`, `warn`, and `error`. The default value is `warn`
+
+* **`--no-restore`**
+
+  Doesn't execute an implicit restore before formatting. Default is to do implicit restore.
+
+* **`--verify-no-changes`**
+
+  Verifies that no formatting changes would be performed. Terminates with a non zero exit code if any files would have been formatted.
+
+* **`--include <INCLUDE>`**
+
+  A space-separated list of relative file or folder paths to include in formatting. All files in the solution or project are formatted if empty.
+
+* **`--exclude <EXCLUDE>`**
+
+  A space-separated list of relative file or folder paths to exclude from formatting.  The default is none.
+
+* **`--include-generated`**
+
+  Formats files generated by the SDK.
+
+* **`-v|--verbosity <LEVEL>`**
+
+  Sets the verbosity level. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`. Default value is `m[inimal]`.
+
+* **`--binarylog <BINARY-LOG-PATH>`**
+
+  Logs all project or solution load information to a binary log file.
+
+* **`--report <REPORT-PATH>`**
+
+  Produces a JSON report in the directory specified by `<REPORT_PATH>`.
+
+* **`-h|--help`**
+
+  Shows help and usage information
+
+## Subcommands
+
+### Whitespace
+
+`dotnet format whitespace` - Formats code to match `editorconfig` settings for whitespace.
```

Comment:
What's the difference between this command and `dotnet format --fix-whitespace`? Might worth adding a note about that.